hooks/index.php

```diff
@@ -4,7 +4,6 @@
 require_once __DIR__ . '/accordion.php';
 require_once __DIR__ . '/search.php';
 require_once __DIR__ . '/pages.php';
-require_once __DIR__ . '/wpsync.php';
```

Comment:
Can we delete that file now?

Reply:
We could, sure. I wasn't 100% sure if we'd like to fall back to using parts of it for specific tasks, but I guess we can always just pull the old commit, right?
I'll remove the file.